lib/services/crawler/crawler_test.go

```diff
@@ -0,0 +1,69 @@
+package crawler_test
+
+import (
+	"github.com/markgravity/golang-ic/database"
+	"github.com/markgravity/golang-ic/lib/models"
+	"github.com/markgravity/golang-ic/lib/services/crawler"
+	"github.com/markgravity/golang-ic/test"
+	"github.com/markgravity/golang-ic/test/fabricators"
+
+	"github.com/go-faker/faker/v4"
+	"github.com/gocolly/colly/v2"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Crawler", func() {
+	AfterEach(func() {
+		test.CleanUpDatabase()
+	})
+
+	Describe("#Run", func() {
+		Context("given VALID keyword", func() {
+			It("returns the parsing result", func() {
```

Comment:
As we have `err := service.Run()`... that suggests `#Run` does not return the parsing result. 🙈 

Based on the `expect` statements, what about `it processes the keyword and updates it in DB`?